app/views/layouts/application.html.haml

```diff
@@ -23,9 +25,10 @@
       .container
         %button.navbar-toggle(type="button" data-toggle="collapse" data-target=".navbar-responsive-collapse")
           %span.icon-bar
-          %span.icon-bar
-          %span.icon-bar
-        %a.navbar-brand(href="/") story.board
+        %img{:src => asset_path("storyboard_logo.png"),:class => "logostyle"}
+        %span.glyphicon.glyphicon-user 
+        %em  Sign in
```

Comment:
In a productive system you would not point to features that are not implemented at all..